plugins/hydra_ray_launcher/example/aws/model/my_model.py

```diff
@@ -0,0 +1,8 @@
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class MyModel:
+    def __init__(self):
+        log.info("Init my model.")
```

Comment:
I like this one much more than `mylib.hello()` :)

It will be good to also save a dummy checkpoint file as a part of the job to see how we handle it.
Where it goes, what can the user do to access it later etc.